Rakefile

```diff
@@ -10,6 +10,13 @@ RSpec::Core::RakeTask.new(:unit) do |t,args|
   t.verbose     = true
 end
 
+desc "Integration tests"
```

Comment:
There isn't any real precedent for segregating this AFAIK (unless you found an example somewhere?), so I'd prefer to just stay consistent with how this works in the other modules:

* https://github.com/puppetlabs/puppetlabs-acl/blob/master/Rakefile
* https://github.com/puppetlabs/puppetlabs-registry/blob/master/Rakefile
* https://github.com/puppetlabs/puppetlabs-reboot/blob/master/Rakefile
* https://github.com/puppetlabs/puppetlabs-sqlserver/blob/master/Rakefile
* https://github.com/puppetlabs/puppetlabs-wsus_client/blob/master/Rakefile